tooling-core/src/main/java/org/opentest4j/reporting/tooling/core/htmlreport/DefaultHtmlReportWriter.java

```diff
@@ -120,9 +120,13 @@ private List<Execution> extractData(List<Path> xmlFiles, Path htmlFile) throws E
 		return executions;
 	}
 
-	private static Path tryRelativize(Path htmlFile, Path xmlFile) {
+	static Path tryRelativize(Path htmlFile, Path xmlFile) {
 		try {
-			return htmlFile.getParent().relativize(xmlFile);
+			var parent = htmlFile.toAbsolutePath().getParent();
+			if (parent == null) {
+				return xmlFile;
```

Comment:
```suggestion
				return htmlFile;
```

Reply:
No, we're trying to relativize the XML file and want to fallback on the provided path if we can't.